scripts/bin/base_qos_init.py

```diff
@@ -660,9 +702,195 @@ def init_default_buffer_pool(pool_type, size):
 
     return buffer_pool_id
 
+# CPS helper functions
+
+def cps_get(obj, qual, data, resp):
+    return (cps.get([cps_object.CPSObject(obj,
+                                          qual=qual,
```

Comment:
Spacing looks all wonky here and the next 2 functions. It seems like you're trying to clearly indicate paren matching? But I suggest compressing this down to 2-3 lines (whatever fits inside 80 columns) and letting text editors indicate paren matching instead.